services/src/main/resources/locale/portlet/news/News_en.properties

```
@@ -192,3 +192,13 @@ news.list.settings.viewTemplate.NewsLatest=Latest News
 news.list.settings.namePlaceholder=Name the News list
 news.list.settings.name.errorMessage=This field must contain only alphanumeric characters
 news.list.settings.drawer.save=Save
+
+newsTargets.settings.title=Manage news publication targets
```

```suggestion
newsTargets.settings.title=Manage news publish targets
```
To be verified with @samehaidi

---

ok i will check with her